### ISSUE TITLE
Request Admin Only Function

### ISSUE THREAD

**Issue by DPxKnightxZero** [elix-x/Teleplates#3]
So I am basically wondering if I can prevent all players except admins from making or placing teleplates. I am thinking though if the got it as a reward etc that would be cool.

**Comment by elix-x** [comment]
In 1.1n, recipe system changed to excore's recipe system. On excore's page it's written how to disable recipe.
Sorry, forgot to write it there, to disable recipe you leave recipe array empty.

Will that work for you, or do you prefer only placable by admins?

**Comment by DPxKnightxZero** [comment]
Well actually some commands like /telepadmin [allow:deny] <dim>:<player>  [craft:place:use] <padname> [true:false]

Did I explain my request clearly enough? I can be a bit more specific. In the long run I would like to be able to change telepad functionality and limitations on the fly. With config settings that is not possible. Um I am hoping some day to run a 1.9 server but right now I am running a 1.7.10 server and considering 1.8.8

**Comment by elix-x** [comment]
Yes you did.
But some teleplates function cannot be changed in runtime...

I'll think about command. Your's is not best design i can think of.

**Comment by DPxKnightxZero** [comment]
Yeah I understand that. Also when you get a chance you should check out the massive performance increase in the 1.8.9 build of latest forge beta gradle. ITS INSANE!

**Comment by elix-x** [comment]
Ok.
Do you want me to update to teleplates to 1.8?
I am going to, but just not right now.

**Comment by DPxKnightxZero** [comment]
Nah just go straight to 1.8.9 it has all of the 1.8 increase and more. Suddenly seems like all mods are jumping to 1.8.9 but I can see why.

**Comment by elix-x** [comment]
Most mods comaptible with 1.8 are compatible with 1.8.9. Not a lot of stuff changed.

**Comment by DPxKnightxZero** [comment]
Hmmm... it makes debugging a lot easier if you have a target version. I will test your content either way and I would of course report bugs. I just know that 1.8 was not that great. Its true 1.8, 1.8.2, 1.8.4 didn't have many changes but if you like at the Minecraft bugfix list 1.8.8 and 1.8.9 have a great deal of bug fixes!

**Comment by DPxKnightxZero** [comment]
I suppose if your attempting to target lowest possible version you could always start at 1.8.8 but the populous is going for 1.8.9 even LexManos himself.

**Comment by elix-x** [comment]
I'll work with 1.8.9. I just told about 1.8 so you know.
Also, i know that lex is for 1.8.9 too.

**Comment by DPxKnightxZero** [comment]
Well thank you very much for your continued efforts! I hope I didn't come off as pushie I was not trying to. :)

**Comment by elix-x** [comment]
No you werent.

To everybody, i'm accepting command suggestions.
